packages/sonataflow-deployment-webapp/pom.xml

```diff
@@ -0,0 +1,132 @@
+    <description>
+        The parent contains all metadata (including plugins) and also dependency versions
+        for all Serverless Workflows modules
+    </description>
+    <inceptionYear>2023</inceptionYear>
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <properties>
+        <java.module.name />
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <maven.compiler.release>11</maven.compiler.release>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <dependencies>
+    </dependencies>
+
+    <repositories>
+        <repository>
+            <id>webjars</id>
+            <url>https://maven.webjars.org</url>
+        </repository>
+    </repositories>
+
+    <distributionManagement>
+        <snapshotRepository>
+            <id>ossrh</id>
+            <url>https://s01.oss.sonatype.org/content/repositories/snapshots</url>
+        </snapshotRepository>
+        <repository>
+            <id>ossrh</id>
+            <url>https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+        </repository>
+    </distributionManagement>
+
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-resources-plugin</artifactId>
+                <version>3.3.1</version>
+                <executions>
+                    <execution>
+                        <id>copy-webjar-resources</id>
+                        <phase>process-resources</phase>
+                        <goals>
+                            <goal>copy-resources</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>${project.build.outputDirectory}</outputDirectory>
+                            <resources>
+                                <resource>
+                                    <directory>dist</directory>
+                                    <includes>
+                                        <include>**/*.html</include>
+                                        <include>**/*.css</include>
+                                        <include>**/*.js</include>
+                                        <include>**/*.svg</include>
+                                        <include>**/*.png</include>
+                                        <include>fonts/**/*</include>
+                                    </includes>
+                                </resource>
+                            </resources>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
```

Comment:
Does this plugin publish to webjars without going through npm?

Reply:
It is used by the command `mvn clean install` which build & installs the package in the local`~/.m2` repository, but doesn't publish anything.
I added this pom.xml only for local tests.

Reply:
Oh ok! I am already thinking about the publishing flow.
My first thought is to add a property in the `package.json` file that tells the package must be published as webjar.
In the release process, we can create a job (after the npm publish job) that filters and publishes to webjars all packages that have this particular property. WDYT?

Reply:
I like the idea of the custom `package.json` property. Is `webjarDeploy: boolean` ok?

Reply:
`webjarDeploy` sounds good :)